lib/py/test/test_thrift_file/TestServer.thrift

```diff
@@ -16,8 +16,12 @@
 # specific language governing permissions and limitations
 # under the License.
 #
- 
+
+struct Message {
+   1: optional string body,
+   2: optional i64 num,
+}
 
 service TestServer{
-	string add_and_get_msg(1:string msg)
+	Message add_and_get_msg(1:Message msg)
```

Comment:
I don't think this change is explained in either the PR description of the ticket?

Reply:
My mistake, this shouldn't have made it to the PR as it is not relevant to this issue.

Reply:
Removed in b2672adfea84051c38b7d124f4599b544d11ebbd